tests/test_download.py

```diff
@@ -0,0 +1,68 @@
+import os
+import pytest
+
+from tests.util import hash
+import toolchest_client as toolchest
+
+toolchest_api_key = os.environ.get("TOOLCHEST_API_KEY")
+if toolchest_api_key:
+    toolchest.set_key(toolchest_api_key)
+
+KRAKEN2_SINGLE_END_HASH = 886254946
+
+
+@pytest.mark.integration
+def test_kraken2_output_manual_download():
```

Comment:
About how long does this test take?

Reply:
Not too much longer than the `kraken2` single-end run (~3 mins). This is basically just the same as the normal `kraken2` single-end run, but downloading the output in 3 different ways after the job finishes.